<?xml version="1.0" encoding="UTF-8"?>
<launch>
    <arg name="lidar_tp" default="/livox/lidar"/>

    <param name="to_show" value="false"/>
    <param name="to_update" value="false"/>

    <node pkg="manual_calibration" name="threadtest" type="threadtest" output="screen">
        <param name="lidar_tp" value="$(arg lidar_tp)" /> 
    </node>

</launch>

